### DISCUSSION TITLE
Filament Email Verification

### DISCUSSION THREAD

**Discussion by sluxzinc92** [filamentphp/filament#9746] [Help] [answered]
Hi, is there a tutorial or guide on how to implement the filament emailverification() and emailverificationprompt? If none, can somebody help me? please.. thanks a lot.

**Comment by devhoussam1998** [comment, marked as answer]
did you tried

```
->id('customer')
...
->emailVerification(EmailVerificationPrompt::class)
```

dont forget to add

implements `MustVerifyEmail` in your **User.php** model

```
use Illuminate\Contracts\Auth\MustVerifyEmail;

class User extends Authenticatable implements MustVerifyEmail
{
  ....
}
```

for more information check https://laravel.com/docs/10.x/verification#model-preparation

**Reply by sluxzinc92** [reply]
it's not sending after clicking resend 
<img width="311" alt="image" src="https://github.com/filamentphp/filament/assets/79960348/1bd97942-fd50-4831-b926-ca7f12e219fe">
this is the error afterwards 
<img width="1207" alt="image" src="https://github.com/filamentphp/filament/assets/79960348/b2ff5e80-3c2f-442f-b1a9-88ff7dca050a">

**Reply by devhoussam1998** [reply]
can you show me what changes you made?

**Reply by devhoussam1998** [reply]
instead of `->emailVerification(EmailVerificationPrompt::class)` use only `->emailVerification()` it works for me

**Reply by sluxzinc92** [reply]
Thanks for the help. It's now working. :) Solved the problem  on SSL operation failed by adding 
<img width="232" alt="image" src="https://github.com/filamentphp/filament/assets/79960348/fee94c79-769e-44d2-ad15-a551fcbb2a1b">
to the mailers => [ smtp => []]; 
Thanks again.

**Reply by mwanjoge** [reply]
I never get this to work on official filament app until i use the starter kit but now am having a fresh filament install i need this feature, when i send via laravel all mail are sent when i send an email verification request no email is sent with no error

**Comment by mwanjoge** [comment]
I never get this to work on official filament app until i use the starter kit but now am having a fresh filament install i need this feature, when i send via laravel all mail are sent when i send an email verification request no email is sent with no error

**Reply by PeterManko** [reply]
If your filament is not sending verification or reset emails, laravel 11 has default `QUEUE_CONNECTION` set to database, sat `QUEUE_CONNECTION=sync` or start the worker responsible.
